website/static/js/share/results.js

```diff
@@ -151,8 +151,10 @@ var Contributor = {
             m('a', {
                 href: '#',
                 onclick: function() {
-                    utils.updateFilter(vm, 'match:contributors.familyName:' + contributor.familyName, true);
                     utils.updateFilter(vm, 'match:contributors.givenName:' + contributor.givenName, true);
```

Comment:
You should add the same constraint for givenName, and if neither are provided, make a filter for just the unparsed name. Long term, we can make that unanalyzed, and require it to be an exact match.